<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTagsManagerDialog</class>
 <widget class="QDialog" name="PTagsManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tag manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../playground.qrc">
    <normaloff>:/images/mail-tagged.png</normaloff>:/images/mail-tagged.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="TagsListWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="addTagPushButton">
           <property name="text">
            <string>&amp;Add tag</string>
           </property>
           <property name="icon">
            <iconset resource="../../playground.qrc">
             <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeTagPushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Remove tag</string>
           </property>
           <property name="icon">
            <iconset resource="../../playground.qrc">
             <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Tag properties</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NameLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../playground.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PTagsManagerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PTagsManagerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addTagPushButton</sender>
   <signal>pressed()</signal>
   <receiver>PTagsManagerDialog</receiver>
   <slot>newTag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeTagPushButton</sender>
   <signal>pressed()</signal>
   <receiver>PTagsManagerDialog</receiver>
   <slot>removeTag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NameLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>PTagsManagerDialog</receiver>
   <slot>setName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newTag()</slot>
  <slot>removeTag()</slot>
  <slot>setName()</slot>
 </slots>
</ui>
